ApollosProject/apollos-prototype
# Live item polling

## DESCRIPTION

### What does this PR do, or why is it needed?

This PR does the following things
- Removes the LiveButton and the polling that occurs with LiveButton.
- Polls for LiveContent, and updates the badge on live content when it becomes live.

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

Okay - this is a tricky multistep process. 

1. Before you boot your server, change the following line to 
```
isLive: true,
```
https://github.com/ApollosProject/apollos-prototype/blob/live-item-polling/packages/apollos-data-connector-church-online/src/data-source.js#L24
2. Boot your server. Make sure to update build before you boot. `yarn && yarn start`
3. The main content item should have a live badge.
4. Change the line you orignally changed to 
```
isLive: false,
```
5. Wait for your server to rebuild (should happen when you save the file) 
6. Wait for the content-item poll to trigger. You should see a log in your server indicating a network request. 
7. The live badge should go away. 

*If this does not work the first time, try it a second time before you pass it off as broken* Sometimes the poll that happens can happen while the server is rebuilding and rebooting, which causes the content-item not to update. 

### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
